WordPress/src/main/java/org/wordpress/android/ui/media/WordPressMediaUtils.java

```diff
@@ -330,4 +346,12 @@ public static void loadNetworkImage(String imageUrl, NetworkImageView imageView,
             imageView.setImageResource(0);
         }
     }
+
+    /**
+     * Returns a poster (thumbnail) URL given a VideoPress video URL
+     * @param videoUrl the remote URL to the VideoPress video
+     */
+    public static String getVideoPressVideoPosterFromURL(String videoUrl) {
+        return videoUrl.substring(0, videoUrl.lastIndexOf(".")) + "_std.original.jpg";
```

Comment:
In addition to the null check mentioned above, this should also ensure `videoUrl` isn't the empty string (otherwise there will be an `StringIndexOutOfBoundsException` exception here). A simple `TextUtils.isEmpty()` check would suffice.